queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py

```diff
@@ -70,13 +70,3 @@ def has_new_nr_for_correction(filing: dict):
             get('nameRequest').get('nrNumber', None)
         return old_nr_number != new_nr_number
     return False
-
-
-def has_new_nr_for_filing(business: Business, filing: dict, filing_type: str):
-    """Return whether a alteration filing has new NR."""
-    nr_number = get_str(filing, f'/filing/{filing_type}/nameRequest/nrNumber')
-    legal_name = get_str(filing, f'/filing/{filing_type}/nameRequest/legalName')
-    if nr_number and legal_name and business.legal_name:
-        # legal api validates legal name in filing json, confirm both are different
-        return legal_name.lower() != business.legal_name.lower()
```

Comment:
By the time call stack reach here business would have already updated with the new legal name. So this will always return false